dev-python/logbook/logbook-1.5.3.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="A logging replacement for Python"
+HOMEPAGE="https://logbook.readthedocs.io/en/stable/
+	https://github.com/getlogbook/logbook
+	https://pypi.org/project/Logbook/"
+SRC_URI="https://github.com/getlogbook/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+distutils_enable_tests pytest
+distutils_enable_sphinx docs
+
+BDEPEND="
+	test? (
+		app-arch/brotli[${PYTHON_USEDEP},python]
+		dev-python/execnet[${PYTHON_USEDEP}]
+		dev-python/jinja[${PYTHON_USEDEP}]
+		dev-python/pyzmq[${PYTHON_USEDEP}]
+		dev-python/sqlalchemy[${PYTHON_USEDEP}]
+	)"
+RDEPEND="!!dev-python/contextvars"
```

Comment:
The blocker needs to be test-time too, obviously. `d_e_t` would take care of adding RDEPEND to test-depends *if* you called it *after* setting RDEPEND.